src/main/app/src/app/shared/material-form-builder/form-components/html-editor/html-editor-variabelen-kies-menu.component.less

```diff
@@ -0,0 +1,8 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+::ng-deep.variabelenMenu {
```

Comment:
ng-deep is ook deprecated, maar wel eens nodig in sommige situaties.

Reply:
Is hier helaas wel nodig, zie ook vorige comment